apex/transformer/tensor_parallel/layers.py

```diff
@@ -520,7 +501,7 @@ def __init__(
         params_dtype=torch.float32,
         use_cpu_initialization=False,
         gradient_accumulation_fusion=False,
-        accumulation_in_fp16: bool = False,
+        accumulation_in_fp16: bool = False, # Deprecated
```

Comment:
just a comment, no need to be accessed.
if this is "deprecated" (well even this transformer module itself is going to be deprecated), I'd recommend making this argument "Optional" and throwing a warning if a value is set

```suggestion
        accumulation_in_fp16: Optional[bool] = None,
```

Reply:
I think this is a nicer approach. I don't see this option used anywhere in Apex, NeMo, or Megatron-LM, so we could even remove it if we're feeling brave.